baybe/recommenders/pure/bayesian/base.py

```diff
@@ -32,6 +33,12 @@ class BayesianRecommender(PureRecommender, ABC):
     )
     """The used acquisition function class."""
 
+    _train_x: Optional[pd.DataFrame] = field(default=None, init=False)
```

Comment:
Hi @Scienfitz, following up yesterday's discussion: As you can see, this implementation stores the transformed data as attributes, similar to the acquisition function. Not sure if it's nice or not, and there are certainly ways to do it without explicitly storing them, but unfortunately not with the our current setup (where bayesian recommender makes a `super` call) – at least I couldn't find any. We can definitely try to change if we want, but I think that would require some additional changes in control flow... Let me know what you think

Reply:
`obsolete after refactoring`